src/tests/test_dc_verifier.cpp

```diff
@@ -0,0 +1,66 @@
+#include <memory>
+#include <string>
+
+#include "algorithms/algo_factory.h"
+#include "all_csv_configs.h"
+#include "dc/verifier/dc_verifier.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "names_and_descriptions.h"
+
+namespace tests {
+
+using namespace algos;
+using namespace algos::dc;
+
+namespace mo = model;
+
+static algos::StdParamsMap GetParamMap(CSVConfig const& csv_config, std::string dc) {
+    using namespace config::names;
+    return {{kCsvConfig, csv_config}, {kDenialConstraint, dc}};
+}
+
+struct DCTestParams {
+    std::string dc_string;
+    CSVConfig csv_config;
+    bool expected;
+};
+
+class TestDCVerifier : public ::testing::TestWithParam<DCTestParams> {};
+
+TEST_P(TestDCVerifier, DefaultTest) {
+    DCTestParams const& p = GetParam();
+    algos::StdParamsMap params = GetParamMap(p.csv_config, p.dc_string);
+    std::unique_ptr<DCVerifier> dc_verifier = algos::CreateAndLoadAlgorithm<DCVerifier>(params);
```

Comment:
nit:
```suggestion
    auto dc_verifier = algos::CreateAndLoadAlgorithm<DCVerifier>(params);
```
As for me, `dc_verifier` type here is obvious